website/docs/d/sm_arbitrary_secret.html.markdown

```diff
@@ -0,0 +1,80 @@
+  instance_id   = "6ebc4224-e983-496a-8a54-f40a0bfa9175"
+  region        = "us-south"
+  id = "0b5571f7-21e6-42b7-91c5-3f5ac9793a46"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `id` - (Required, Forces new resource, String) The ID of the secret.
+  * Constraints: The maximum length is `36` characters. The minimum length is `36` characters. The value must match regular expression `/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/`.
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the ArbitrarySecret.
+* `created_at` - (String) The date when a resource was created. The date format follows RFC 3339.
+
+* `created_by` - (String) The unique identifier that is associated with the entity that created the secret.
+  * Constraints: The maximum length is `128` characters. The minimum length is `4` characters.
+
+* `crn` - (String) A CRN that uniquely identifies an IBM Cloud resource.
+  * Constraints: The maximum length is `512` characters. The minimum length is `9` characters. The value must match regular expression `/^crn:v[0-9](:([A-Za-z0-9-._~!$&'()*+,;=@\/]|%[0-9A-Z]{2})*){8}$/`.
+
+* `custom_metadata` - (Map) The secret metadata that a user can customize.
+
+* `description` - (String) An extended description of your secret.To protect your privacy, do not use personal data, such as your name or location, as a description for your secret group.
+  * Constraints: The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/(.*?)/`.
+
+* `downloaded` - (Boolean) Indicates whether the secret data that is associated with a secret version was retrieved in a call to the service API.
+
+* `expiration_date` - (Forces new resource, String) The date a secret is expired. The date format follows RFC 3339.
+
+* `labels` - (List) Labels that you can use to search for secrets in your instance.Up to 30 labels can be created.
+  * Constraints: The list items must match regular expression `/(.*?)/`. The maximum length is `30` items. The minimum length is `0` items.
+
+* `locks_total` - (Integer) The number of locks of the secret.
+  * Constraints: The maximum value is `1000`. The minimum value is `0`.
+
+* `name` - (String) The human-readable name of your secret.
+  * Constraints: The maximum length is `256` characters. The minimum length is `2` characters. The value must match regular expression `/^\\w(([\\w-.]+)?\\w)?$/`.
+
+* `payload` - (Forces new resource, String) The arbitrary secret's data payload.
+  * Constraints: The maximum length is `100000` characters. The minimum length is `0` characters. The value must match regular expression `/(.*?)/`.
+
+* `secret_group_id` - (Forces new resource, String) A v4 UUID identifier, or `default` secret group.
```

Comment:
Datasources and Attribute references (computed atributes) shouldn't hold the word "Forces new resource"